grails-app/domain/mama/ng/scheduler/Schedule.groovy

```diff
@@ -0,0 +1,30 @@
+package mama.ng.scheduler
+
+class Schedule {
+
+    static hasMany = [messages: Message]
+
+    String id = UUID.randomUUID().toString()
+    String userId
```

Comment:
Hmm. Two thoughts:

1. Wondering whether we should replace userId with some generic way to
specify additional data to send along with the HTTP request made by the
scheduler? That's probably better left for a later PR though.

2. Even if we have this additional data, I guess userId would be fairly
useful as a way to track the person (or other entity) the schedule is
associated with.

3. What about ``receiverId``? (as in, the id of the person the schedule
events are for?).